packages/radix-ui-themes/src/components/reset.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Slot } from '@radix-ui/react-slot';
+import { extractProps, marginPropDefs, requireReactElement } from '../helpers';
+
+import type { MarginProps, PropsWithoutRefOrColor } from '../helpers';
+
+type ResetElement = React.ElementRef<'div'>;
+interface ResetProps extends PropsWithoutRefOrColor<typeof Slot>, MarginProps {}
```

Comment:
Would having MarginProps be a nice DX to help you build custom components on top of Reset?

Reply:
That's nice yeah I like that.